src/application_input.hpp

```diff
@@ -36,6 +36,7 @@ struct ApplicationInput {
   std::function<void(Mesh *, ParameterInput *)> InitUserMeshData = nullptr;
   std::function<void(Mesh *, ParameterInput *, MeshData<Real> *)> MeshProblemGenerator =
       nullptr;
+  std::function<void(Mesh *, MeshData<Real> *)> MeshPostInitialization = nullptr;
```

Comment:
Does one need to pass both `Mesh` and `MeshData`? I suppose they could have different subsets of blocks

Reply:
I myself would certainly never use `PostInitialization` in a way that would *require* both `Mesh` and `MeshData`.  Here I just tried to keep the function arguments consistent between the `Mesh` variations of `ProblemGenerator` and `PostInitialization`.